docs/foundation/members.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar: true
+tagline: "Foundation"
+description: "Information about the Bitcoin Dev Kit Foundation"
+editLink: false
+lastUpdated: false
+---
+
+# Members
+
+The Bitcoin Dev Kit project is proudly supported by a number of members.
+
+<div style="text-align: left; margin-bottom: 2rem;">
+  <a href="/foundation/become-a-member" style="display: inline-block; background-color: var(--docs-primary-dark); color: var(--docs-dark); padding: 10px 20px; text-decoration: none; border-radius: 5px; font-weight: bold;">Become A Member</a>
+</div>
+
+## Platinum
+
+<div class="members">
+  <div class="row">
+      <a href="https://spiral.xyz/" target="_blank" rel="nofollow noopener">
+          <img src="/img/spiral.svg" alt="Sponsor Spiral">
+          Spiral
+      </a>
+  </div>
+</div>
+<br>
+
+## Gold
+
+<div class="members">
+  <div class="row">
+      <a href="https://opensats.org/"  target="_blank" rel="nofollow noopener">
+        <img src="/img/opensats.png" alt="OpenSats">
+        OpenSats
+      </a>
+  </div>
+</div>
+<br>
+
+## Silver
+
+<div class="members">
+  <div class="row">
+      <a href="https://spiral.xyz/" target="_blank" rel="nofollow noopener">
+          <img src="/img/spiral.svg" alt="Sponsor Spiral">
+          Spiral
+      </a>
+      <a href="https://opensats.org/"  target="_blank" rel="nofollow noopener">
+        <img src="/img/opensats.png" alt="OpenSats">
+        OpenSats
+      </a>
```

Comment:
These are already listed a platinum and gold so shouldn't be listed under silver. I suspect you just put these here to test the formatting. I do like how it looks running horizontally.

Reply:
Updated.